README.md

```diff
@@ -198,530 +116,87 @@ Then python client will switch its endpoint to the one specified by the values o
-```
-
-```python
-els = self.driver.find_elements_by_ios_predicate('wdValue == "SearchBar" AND isWDDivisible == 1')
-self.assertIsInstance(els, list)
-```
-
-
-### Finding elements by iOS class chain
-
-**This method is only for [XCUITest driver](https://github.com/appium/appium-xcuitest-driver)**
-
-This method allows finding elements using iOS class chain. The methods take
-a string in the format of a class chain, including element type.
-
-Adds the methods
-`driver.find_element_by_ios_class_chain` and `find_elements_by_ios_class_chain`.
-
-```python
-el = self.driver.find_element_by_ios_class_chain('XCUIElementTypeWindow/XCUIElementTypeButton[3]')
-self.assertIsNotNone(el)
-```
-
-```python
-els = self.driver.find_elements_by_ios_class_chain('XCUIElementTypeWindow/XCUIElementTypeButton')
-self.assertIsInstance(els, list)
-```
-
-### Finding elements by Accessibility ID
-
-Allows for elements to be found using the "Accessibility ID". The methods take a
-string representing the accessibility id or label attached to a given element, e.g., for iOS the accessibility identifier and for Android the content-description. Adds the methods
-`driver.find_element_by_accessibility_id` and `find_elements_by_accessibility_id`.
-
-```python
-el = self.driver.find_element_by_accessibility_id('Animation')
-self.assertIsNotNone(el)
-```
-
-```python
-els = self.driver.find_elements_by_accessibility_id('Animation')
-self.assertIsInstance(els, list)
-```
-
-
-### Touch actions
-
-In order to accommodate mobile touch actions, and touch actions involving
-multiple pointers, the Selenium 3.0 draft specifies ["touch gestures"](https://dvcs.w3.org/hg/webdriver/raw-file/tip/webdriver-spec.html#touch-gestures) and ["multi actions"](https://dvcs.w3.org/hg/webdriver/raw-file/tip/webdriver-spec.html#multiactions-1), which build upon the touch actions.
-
-move_to: note that use keyword arguments if no element
-
-The API is built around `TouchAction` objects, which are chains of one or more actions to be performed in a sequence. The actions are:
-
-#### `perform`
-
-The `perform` method sends the chain to the server in order to be enacted. It also empties the action chain, so the object can be reused. It will be at the end of all single action chains, but is unused when writing multi-action chains.
-
-#### `tap`
-
-The `tap` method stands alone, being unable to be chained with other methods. If you need a `tap`-like action that starts a longer chain, use `press`.
-
-It can take either an element with an optional x-y offset, or absolute x-y coordinates for the tap, and an optional count.
-
-```python
-el = self.driver.find_element_by_accessibility_id('Animation')
-action = TouchAction(self.driver)
-action.tap(el).perform()
-el = self.driver.find_element_by_accessibility_id('Bouncing Balls')
-self.assertIsNotNone(el)
-```
-
-#### `press`
-
-#### `long_press`
-
-#### `release`
-
-#### `move_to`
+https://python-client-sphinx.github.io/ is detailed documentation
```

Comment:
Here will be https://appium.github.io/python-client-sphinx/ .
(e.g. https://ki4070ma.github.io/python-client-sphinx/ )

[Notes] I'd recommend to change repo name to better one, it doesn't need to be in this PR, though. Current name is just copied from my temporary one.
At least `sphinx` is unnecessary. (e.g. https://appium.github.io/python-client-doc/ )

Reply:
Okay. Np to replace it later. We can update here then.